private-templates-service/client/src/App.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { Component } from "react";
+import { BrowserRouter as Router, Route } from "react-router-dom";
+import { Container } from "reactstrap";
+
+import { UserAgentApplication, ClientAuthError } from "msal";
+
+import NavBar from "./components/NavBar/NavBar";
+import ErrorMessage from "./components/ErrorMessage/ErrorMessage";
+import Welcome from "./components/Welcome/Welcome";
+import config from "./Config";
+import { getUserDetails, getOrgDetails } from "./Services/GraphService";
+import { ErrorMessageProps } from "./components/ErrorMessage/ErrorMessage";
+
+import "bootstrap/dist/css/bootstrap.css";
+
+export interface UserType {
+  displayName?: string;
```

Comment:
Is it possible for a user to be logged in, and not have these values? If not lets make them required so we don't need to do null checks and stuff later on